source/reference/minio-mc-admin/mc-admin-cluster-iam-import.rst

```diff
@@ -21,6 +21,33 @@ The :mc:`mc admin cluster iam import` command imports :ref:`IAM <minio-authentic
 
 You can use this command to manually restore IAM metadata settings for a MinIO deployment.
 
+.. versionadded:: mc RELEASE.2024-09-09T07-53-10Z
+
+   The command outputs the results of the import, including the following:
+
+   - number of indidividual entities imported:
+    
+     - policies
+     - users
+     - groups
+     - service accounts
+
+   - policies imported to a:
+
+     - user
+     - group
+     - service account
```

Comment:
Like a complete list of policies per user/group/sa or just a count?

Reply:
List. Clarified.